integrations/azure_ai_search/src/haystack_integrations/document_stores/azure_ai_search/document_store.py

```diff
@@ -435,6 +440,6 @@ def _embedding_retrieval(
             raise ValueError(msg)
 
         vector_query = VectorizedQuery(vector=query_embedding, k_nearest_neighbors=top_k, fields="embedding")
-        result = self.client.search(search_text=None, vector_queries=[vector_query], select=fields, filter=filters)
+        result = self.client.search(vector_queries=[vector_query], filter=filters, **kwargs)
```

Comment:
why `select=fields` has been removed?

Reply:
It can be passed as in `kwargs`, so we don't need to explicitly add it.